Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -461,6 +466,66 @@ def add_comment(self, issue, text):
 
         return result
 
+    def issues_to_ids(self, issues):
+        if not isinstance(issues, list):
+            raise TypeError('Input should be a list of Issue objects.')
+        issue_ids = []
+        for i in issues:
+            try:
+                issue_id = i.link
+            except AttributeError:
+                raise AttributeError('Input should be a list of Issue objects.')
+            if isinstance(i.tracker, Tracker):
```

Comment:
This check should be sufficient, because every issue which has a Bugzilla Tracker will have an ID. And if the caller provides something other than an `issue` object, you'll end up with an `AttributeError` anyways. All of this is to say, I don't think you need the `i.link` check.